src/components/components/dataDisplay/chips/Chip.vue

```diff
@@ -1,7 +1,15 @@
 <template>
 	<span
 		class="c-application c-chip"
-		:class="[computedType, computedColor, computedSize, computedFull, computedTransparent, computedClickable]"
+		:class="[
+			computedType,
+			computedColor,
+			computedSize,
+			computedFull,
+			computedTransparent,
+			computedClickable,
+			{ 'c-chip--with-close-button': withCloseButton },
+		]"
```

Comment:
class를 computed에 몰면 template쪽이 더 깔끔해지겠네요
```suggestion
		:class="classes"
```

```
computed: {
    classes() {
        return [
	    computedType,
    	    computedColor,
	    computedSize,
	    computedFull,
	    computedTransparent,
	    computedClickable,
	    { 'c-chip--with-close-button': withCloseButton },
        ];
    }
}
```